include/sunlinsol/sunlinsol_spgmr.h

```diff
@@ -29,33 +29,33 @@
 #define _SUNLINSOL_SPGMR_H
 
 #include <stdio.h>
-
 #include <sundials/sundials_linearsolver.h>
 #include <sundials/sundials_matrix.h>
 #include <sundials/sundials_nvector.h>
 
-#ifdef __cplusplus  /* wrapper to enable C++ usage */
+#ifdef __cplusplus /* wrapper to enable C++ usage */
 extern "C" {
 #endif
 
 /* Default SPGMR solver parameters */
-#define SUNSPGMR_MAXL_DEFAULT    5
-#define SUNSPGMR_MAXRS_DEFAULT   0
-#define SUNSPGMR_GSTYPE_DEFAULT  SUN_MODIFIED_GS
+#define SUNSPGMR_MAXL_DEFAULT   5
+#define SUNSPGMR_MAXRS_DEFAULT  0
+#define SUNSPGMR_GSTYPE_DEFAULT SUN_MODIFIED_GS
 
 /* ----------------------------------------
  * SPGMR Implementation of SUNLinearSolver
  * ---------------------------------------- */
 
-struct _SUNLinearSolverContent_SPGMR {
+struct _SUNLinearSolverContent_SPGMR
+{
   int maxl;
   int pretype;
   int gstype;
   int max_restarts;
   booleantype zeroguess;
   int numiters;
   realtype resnorm;
-  int last_flag;
+  SUNLsStatus last_flag;
```

Comment:
```suggestion
  int last_flag;
```
All of the other iterative linear solvers retained `int` for their internal `last_flag` parameter except this one (and `SUNLinSolLastFlag_SPGMR` returns a `sunindextype`)